design-editor/src/panel/preview/preview-element-vd.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-disable no-console */
+/* eslint-disable max-len */
+function startVD (profile) {
+	let stDevice, stProperty, stValue, eventType;
+	const previewEl = document.querySelector('.closet-preview-frame').contentWindow;
+	const tau = previewEl.tau;
+
+	/* iFrame Element */
+	const textData = previewEl.document.querySelector('input[type=\'text\']');
```

Comment:
Why we take care of first input element only? What if there are multiple elements?

Reply:
We implemented the implementation based on TC.
TC adds Dimmer, Graph, and Input (text or slider or both) one by one to test the operation.
We plan to modify it to control several elements later.